android/app/src/main/java/com/zulipmobile/notifications/FCMPushNotifications.kt

```diff
@@ -82,9 +105,19 @@ internal fun onReceived(context: Context, mapData: Map<String, String>) {
     }
 }
 
+/** Handle a RemoveFcmMessage, removing notifications from the UI as appropriate. */
```

Comment:
I tried the `///`, and the IDE didn't seem to recognize it -- I hovered over a place where the function was used, and the text didn't show up, whereas it did if I used `/** … */`.

I'd hoped to use `///` because it's more compact when these go over a line -- no leading `/**` line and trailing ` */` line with nothing else on them. But alas.